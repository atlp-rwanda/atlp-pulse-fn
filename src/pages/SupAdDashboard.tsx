import React from 'react';
import { useTranslation } from 'react-i18next';
import Chart from '../components/Chart';
import Card from '../components/Card';
import useDocumentTitle from '../hook/useDocumentTitle';
import Comingsoon from './Comingsoon';

function SupAdDashboard() {
  useDocumentTitle('Dashboard');
  const { t } = useTranslation();
  return (
    <div className="flex flex-col grow bg-light-bg dark:bg-dark-frame-bg">
      <div className="flex flex-row pb-8 justify-center">
        <div className="">
          {/* <div className="grid grid-cols-2 lg:grid-cols-4">
            <Card text={t('All Organizations')} number={24} />
            <Card text={t('All Admins')} number={24} />
            <Card text={t('Domains')} number={24} />
            <Card text={t('Income')} number={`$${1000}`} />
          </div>
          <Chart title="Organizations" /> */}

          <Comingsoon title="Dashboard" />
        </div>
      </div>
    </div>
  );
}

export default SupAdDashboard;
