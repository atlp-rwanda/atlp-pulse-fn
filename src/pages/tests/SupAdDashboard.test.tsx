import React from 'react';
import renderer from 'react-test-renderer';
import '../../../test/jest/__mocks__/matchMedia';
import SupAdDashboard from '../SupAdDashboard';

describe('Super Admin Dashboard test ', () => {
  it('Should render SupAdDashboard', () => {
    const elem = renderer.create(<SupAdDashboard />).toJSON();
    expect(elem).toMatchSnapshot();
  });
});
