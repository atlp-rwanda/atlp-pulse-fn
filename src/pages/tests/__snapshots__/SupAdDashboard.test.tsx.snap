// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Super Admin Dashboard test  Should render SupAdDashboard 1`] = `
<div
  className="flex flex-col grow bg-light-bg dark:bg-dark-frame-bg"
>
  <div
    className="flex flex-row pb-8 justify-center"
  >
    <div
      className=""
    >
      <div
        className="flex items-center justify-center h-fit bg-light-bg dark:bg-dark-frame-bg py-4"
      >
        <div
          className="p-5 mx-2 bg-indigo-100 rounded-lg shadow-lg dark:bg-dark-bg md:p-20"
        >
          <div
            className="text-center"
          >
            <h2
              className="text-4xl font-extrabold leading-10 tracking-tight text-light-text dark:text-dark-text-fill sm:text-5xl sm:leading-none md:text-6xl"
            >
              Dashboard
            </h2>
            <h3
              className="mt-10 text-xl text-light-text dark:text-dark-text-fill md:text-3xl"
            >
              comingsoon
            </h3>
            <p
              className="mt-10 text-md md:text-xl text-light-text dark:text-dark-text-fill"
            >
              Somethingnewiscoming
            </p>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
